pom.xml

```diff
@@ -22,14 +22,14 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>dynamic-spark</artifactId>
-  <version>2.2.2</version>
+  <version>2.3.0-SNAPSHOT</version>
 
   <properties>
     <!-- properties for script build step that creates the config files for the artifacts -->
     <data.pipeline.parent>system:cdap-data-pipeline[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT)</data.pipeline.parent>
     <data.stream.parent>system:cdap-data-streams[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT)</data.stream.parent>
 
-    <cdap.version>6.1.3-SNAPSHOT</cdap.version>
+    <cdap.version>6.2.0</cdap.version>
```

Comment:
why is this change needed?

Reply:
6.2.0 was the version in the merged PR the cherrypick is from? Is this incorrect and needs to be changed

Reply:
Also lets make this 6.1.2